### PR TITLE
If applied this commit will add one click way to delete a app passing…

### PR DESCRIPTION
**Pull request by alansilva-luizalabs**
Add one click way to delete a app passing --confirm flag;

### CODE REVIEW

File: pkg/client/cmd/app.go [hunk 1]
```diff
@@ -265,22 +265,34 @@ func appDel(cmd *cobra.Command, args []string) {
...
-		color.YellowString(currentClusterName),
-	)
-	s, _ := client.GetInput(inputMsg)
-	if s != "yes" {
-		fmt.Println("Delete process aborted!")
-		return
+
+	confirm, err := cmd.Flags().GetString("confirm")
+	if err != nil {
+		client.PrintErrorAndExit("Invalid app name")
```
Comment (yagonobre) [inline pkg/client/cmd/app.go]: `Invalid confirm parameter`

File: pkg/client/cmd/app.go [hunk 2]
```diff
@@ -265,22 +265,34 @@ func appDel(cmd *cobra.Command, args []string) {
...
+			fmt.Println("Delete process aborted!")
+			return
+		}
+
+		resp, _ := client.GetInput("Please re type the app name: ")
+		if resp != name {
+			fmt.Println("Delete process aborted!")
+			return
+		}
+	} else if confirm != name {
```
Comment (yagonobre) [inline pkg/client/cmd/app.go]: You can remove this else.

File: pkg/client/cmd/app.go (hunk 1 above)
Comment (yagonobre) [reply pkg/client/cmd/app.go]: Also should it be a boolean?

Comment (vinicyusmacedo) [reply pkg/client/cmd/app.go]: From what I understand, this `--confirm` flag will actually receive the app name. 

I'm not sure if passing the app name is a good approach. For example, if you use the `--force` flag on git, you're probably aware that the action you're about to do might be destructive.

Comment (aguerra) [reply pkg/client/cmd/app.go]: I think `--force` is much clearer (maybe `--silent`).

**Comment by aguerra** [comment]
Watch out for uploading binaries and if possible don't use @alansilva-luizalabs, use your personal github account as you might be removed from Contributors in the future (true story bro).

**Comment by alansilva-luizalabs** [comment]
Hi @yagonobre  @aguerra @vinicyusmacedo. I will open other Pull Request with all suggestions. Thanks.

**Pull request closed without merging**